File: programmers/level1/01_2016년.java
import java.util.*;

class Main {
    public static String solution(int a, int b) {
        String answer = "";
        String[] dateName = {"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};
        int[] dayNumOFMonth = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        int totalOfDayNum = 0;

        for(int i=0; i<a-1; i++){
            totalOfDayNum += dayNumOFMonth[i];
        }

        totalOfDayNum += (b-1);
        answer = dateName[totalOfDayNum%7];

        return answer;
    }

    public static void main(String[] args){
        Main method = new Main();
        System.out.println(method.solution(5,24));
    }
}

File: programmers/level1/01_2016년.py
def solution(a, b):
    answer = ''

    days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    dates = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    tot_days = 0

    for i in range(0, a - 1):
        tot_days += days[i]

    tot_days += b

    answer = str(dates[tot_days % 7 - 1])

    return answer


s = solution(5, 24)
print(s)
